.env.ci

```diff
@@ -0,0 +1,16 @@
+COMPOSE_PROJECT_NAME=elife-xpub
+IMAGE_TAG=develop
+PORT=3000
+
+# connection details for tests
+# TODO: confusing, should just be PGHOST?
+PGHOSTADDR=postgres
+PGHOST=postgres
```

Comment:
`PGHOSTADDR` was added in June last year a commit entitled `fix(ci): change PG_HOST for k8s`. I reckon it should be safe to remove it here.